dotcom-rendering/src/web/components/SecureSignupIframe.importable.tsx

```diff
@@ -0,0 +1,368 @@
+import { css } from '@emotion/react';
+import { neutral, space, until } from '@guardian/source-foundations';
+import {
+	Button,
+	InlineError,
+	InlineSuccess,
+	Link,
+	SvgReload,
+	SvgSpinner,
+} from '@guardian/source-react-components';
+import type { ReactEventHandler } from 'react';
+import { useEffect, useRef, useState } from 'react';
+import ReCAPTCHA from 'react-google-recaptcha';
+import type { OphanAction } from '../browser/ophan/ophan';
+import {
+	getOphanRecordFunction,
+	submitComponentEvent,
+} from '../browser/ophan/ophan';
+
+// The Google documentation specifies that if the 'recaptcha-badge' is hidden,
+// their T+C's must be displayed instead. While this component hides the
+// badge, its parent must include the T+C along side it.
+// The T+C are not included in this componet directly to reduce layout shift
+// from the island hydrating (placeholder height for the text can't
+// be accurately predicated for every breakpoint).
+// https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed
+
+type Props = {
+	styles: string;
+	html: string;
+	newsletterId: string;
+	successText: string;
+};
+
+const ErrorMessageWithAdvice = (props: { text?: string }) => (
```

Comment:
Does this work? `({ text }: { text?: string})` then `... {text} Please try again or contact ...`

Reply:
yes - will change